packages/react-component-library/src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+export const ButtonGroup: React.FC = ({ children }) => (
```

Comment:
Is there an opportunity to tighten this up with an interface as `children` will always be a collection of `Button`?

Reply:
Could do. Whats the syntax for that?

Reply:
NVM if other comments are correct from design then I may have to write this a different way